docs/modules/ROOT/pages/installation.adoc

```diff
@@ -22,6 +22,11 @@ Once you hav installed from one of the below approaches it is recommended you ru
  your `PATH` to include jbang app scripts + it will on operating systems that supports
 it setup a `j!` alias you can use instead of `jbang`.
 
+[NOTE]
```

Comment:
I think this is a bit too "prominent". We should probably find a better place for this, wdyt @maxandersen ?

Reply:
We have a cache section - would put it there.

Reply:
Good idea. I'll move it to the caching.adoc and write up a common text for JBANG_CACHE_DIR and JBANG_DIR . They are related after all.
But the the part hinting that JBANG_DIR is also used for some configurations is still valid for installation.adoc I think, or possible some other more relevant page?